drools-wb-screens/drools-wb-scorecard-xls-editor/drools-wb-scorecard-xls-editor-backend/src/main/java/org/drools/workbench/screens/scorecardxls/backend/server/ScoreCardXLSServiceImpl.java

```diff
@@ -249,15 +247,24 @@ public Path copy( final Path path,
     @Override
     public List<ValidationMessage> validate( final Path path,
                                              final Path resource ) {
+        InputStream inputStream = null;
```

Comment:
I was thinking about that, but (from a brief look) it seems we wouldn't be able to do
throw ExceptionUtilities.handleException( ioe );
so I rather kept it as it is.